lib/manager/maven/extract.js

```diff
@@ -91,13 +95,31 @@ function applyProps(dep, props) {
   return result;
 }
 
-function extractDependencies(raw) {
+function resolveParentFile(packageFile, parentPath) {
+  let parentFile = 'pom.xml';
+  let parentDir = parentPath;
+  const parentBasename = path.basename(parentPath);
+  if (parentBasename === 'pom.xml' || /\.pom\.xml$/.test(parentBasename)) {
+    parentFile = parentBasename;
+    parentDir = path.dirname(parentPath);
+  }
+  const dir = path.dirname(packageFile);
+  return path.normalize(path.join(dir, parentDir, parentFile));
+}
+
+function extractDependencies(raw, packageFile) {
   if (!raw) return null;
 
   const project = parsePom(raw);
   if (!project) return null;
 
-  const result = { datasource: 'maven' };
+  const result = {
+    datasource: 'maven',
+    manager: 'maven',
```

Comment:
It's refactoring, we were doing this one fn call higher

Reply:
manager should be set to maven before this extract function is called. i.e. it's called *because* manager=maven. So that's why I doubt if it's needed.